src/viem/saga.ts

```diff
@@ -306,6 +307,11 @@ export function* sendAndMonitorTransaction({
         transactionHash: receipt.transactionHash,
         block: receipt.blockNumber.toString(),
         status: true,
+        fee: {
+          effectiveGasPrice: receipt.effectiveGasPrice.toString(),
+          gasUsed: receipt.gasUsed.toString(),
+          feeCurrency: 'feeCurrency' in receipt ? receipt.feeCurrency : undefined,
```

Comment:
hey @jeanregisser @bakoushin do you guys have any tips for getting the feeCurrency to show up in the tx receipt? this is a [transaction](https://explorer.celo.org/mainnet/tx/0x3dea919b3a54650350833d59106a797a7fb3e8894e2f96a64394846cf8fc81d5) that i tried, i was expecting the fee currency to be cEUR here but it was undefined :/